exercism_test_helper/lib/test_source/transformer.ex

```diff
@@ -46,4 +47,12 @@ defmodule TestSource.Transformer do
       IO.puts("[test started] test " <> unquote(name))
     end
   end
+
+  defp remove_pending_tag({name, meta, args}) when is_list(args),
+    do: {name, meta, Enum.reject(args, &pending_tag?/1)}
+
+  defp remove_pending_tag(node), do: node
+
+  defp pending_tag?({:@, _, [{:tag, _, [:pending]}]}), do: true
```

Comment:
I _think_ `ExUnit.configure(exclude: :pending, trace: true)` will also exclude tests where the `:pending` tag was defined with some value (also value different than `true`), e.g. `@tag pending: 3`. For completeness, we should strip that too. I would suggest adding a test in `transformer_test.exs` like this (currently failing):

```elixir
  describe "testsuite - more tests" do
    test "strips pending tags" do
      defsuite do
        test "will be run 1" do
          assert(true)
        end

        @tag :pending
        test "will be run 2" do
          assert(true)
        end

        @tag pending: true
        test "will be run 3" do
          assert(true)
        end

        @tag :pending
        @tag :slow
        test "will not be run 1" do
          assert(true)
        end

        @tag :slow
        test "will not be run 2" do
          assert(true)
        end
      end

      desired = """
      [test started] test will be run 1
      [test started] test will be run 2
      [test started] test will be run 3
      """

      # add exclude: :pending to mimic the test_helpers.exs files in every exercise
      output = capture_io(fn -> run([exclude: [:slow, :pending]]) end)

      assert output == desired
    end
  end
```